cogs/users.py

```diff
@@ -61,6 +61,9 @@ async def add(self, interaction: discord.Interaction, leetcode_username: str, in
         user_exists = await User.find_one(User.id == user_id).project(IdProjection)
 
         if user_exists:
+            # It has passed the checks so it will or has been added - add connected role to user
+            await add_verified_role_to_user(interaction)
+
             display_information_exists = await User.find_one(User.id == user_id, User.display_information.server_id == server_id)
```

Comment:
I'm unsure if this is the right way of doing it. I'll have a check when I get home.